src/main/java/seedu/address/model/recipe/ServingSize.java

```diff
@@ -10,7 +10,8 @@
  */
 public class ServingSize {
     public static final String MESSAGE_CONSTRAINTS =
-            "ServingSize size should not be left blank or less than or equals to 0";
+            "ServingSize size should be a valid, reasonable amount of servings that's greater than 0 "
+                    + "and less than 1000 servings (that's literally impossible)";
```

Comment:
Maybe saying "(that's literally impossible)" is unnecessary and appears too confrontational. Better to explain in the UG that it's a limitation of our application (To protect itself) rather than a limitation of what's physically possible.

Reply:
The error message lgtm ah, if im the user, ill laugh. 🌮

Reply:
Can be interpreted either way tbh. I would love it if the users found it funny too but I also can't 100% say that they'll have the same humour

Reply:
Removed it just in case poeple get offended! (buncha pussies)